shiny/global-sensitivity/helper.R

```diff
@@ -0,0 +1,136 @@
+library(shiny)
+library(PEcAn.DB)
+library(RPostgreSQL)
+library(udunits2)
+library(lubridate)
+library(dplyr)
+
+options(scipen=12)
+
+betyConnect <- function(php.config="../../web/config.php") {
```

Comment:
If there are a bunch of helper functions that are the same between the different Shiny apps then we should migrate them out of here and Rob's app and into a package (e.g. visualization)